content/src/denylist/ActiveDenylist.ts

```diff
@@ -73,22 +73,31 @@ export class ActiveDenylist extends Denylist {
       },
       { priority: DB_REQUEST_PRIORITY.HIGH }
     )
-    this.empty = (await this.repository.run((db) => db.denylist.getAllDenylistedTargets())).length === 0
+    this.empty =
+      (
+        await this.repository.run((db) => db.denylist.getAllDenylistedTargets(), {
+          priority: DB_REQUEST_PRIORITY.HIGH
+        })
+      ).length === 0
     return { status: DenylistOperationStatus.OK, type: DenylistValidationType.SIGNATURE_VALIDATION }
   }
 
   async getAllDenylistedTargets(): Promise<{ target: DenylistTarget; metadata: DenylistMetadata }[]> {
     if (this.empty) {
       return []
     }
-    return this.repository.run((db) => db.denylist.getAllDenylistedTargets())
+    return this.repository.run((db) => db.denylist.getAllDenylistedTargets(), {
+      priority: DB_REQUEST_PRIORITY.LOW
```

Comment:
Why is this low? Isn't this used when fetching content or something like that?

Reply:
Not really, the method `getAllDenylistedTargets` is only used when fetching `/denylist` that lists every element in the denylist. To check if content should be retrieved we are using: https://github.com/decentraland/catalyst/pull/649/files#diff-dd8d71d82d98295be817d09a1d0edc30cc893b9611b53511e48b7b6e5d4508c7R57